app/views/assessments/assign.html.erb

```diff
@@ -69,6 +71,10 @@
                     <td>
                       <%= need_fields.select :status, need_statuses, {}, { class: "dropdown", "data-minimum-results-for-search" => "Infinity" } %>
                     </td>
+                    <td>
+                      <%= need_fields.check_box :send_email, {class: 'bulk-action-checkbox', checked: false,  id: "send-email"}, 'true', 'false' %>
```

Comment:
Given the other conversation then - should the checked value be read here, to keep previous selections?

Reply:
yes in this case